src/c2_blockchain/p4_batched_extrinsics.rs

```diff
@@ -16,7 +16,7 @@ pub struct Header {
     // For example, a hash or a Merkle root.
     extrinsics_root: Hash,
     state: u64,
-    consensus_digest: u64,
+    pub consensus_digest: u64,
```

Comment:
Why does this need to be `pub`? Especially if the other fields on the same struct don't need to be?

Reply:
Because of the mine_extra_hard() in p5 where we have to change the consensus_digest